pkg/remotewrite/prometheus.go

```diff
@@ -34,11 +34,14 @@ func MapTagSet(t *metrics.TagSet) []*prompb.Label {
 //
 // The labels are lexicographic sorted as required
 // from the Remote write's specification.
-func MapSeries(series metrics.TimeSeries) []*prompb.Label {
+func MapSeries(series metrics.TimeSeries, suffix string) []*prompb.Label {
 	lbls := append(MapTagSet(series.Tags), &prompb.Label{
 		Name:  namelbl,
 		Value: fmt.Sprintf("%s%s", defaultMetricPrefix, series.Metric.Name),
 	})
+	if suffix != "" {
+		lbls[len(lbls)-1].Value += "_" + suffix
```

Comment:
:thinking: Out of curiosity, do we need to set it on an array with `lbls[len(lbls)-1]` ?

Could it be more straightforward, something like:
```
v := defaultMetricPrefix + series.Metric.Name
if suffix != "" {
    v+= suffix
}

...
// use v